index.bs

```diff
@@ -1919,7 +1919,9 @@ attObj = {
 
 attStmtTemplate = (
                       fmt: text,
-                      attStmt: bytes
+                      attStmt: {
```

Comment:
I think [CDDL](https://tools.ietf.org/html/draft-ietf-cbor-cddl-00#page-18) says this should be 
```cddl
  attStmt: { * tstr => any }
```

It could be `* any=>any`, but all the formats I see use just strings as the keys, and using multiple types for keys makes canonicalization a bit more difficult, so I agree with restricting it to just `tstr`.